ios/NotificationService/NotificationService.swift

```diff
@@ -9,6 +9,11 @@ class NotificationService: UNNotificationServiceExtension {
 
   var retryIndex = 0
   
+  override init() {
+    super.init()
+    initSentryAppExt()
+  }
+  
```

Comment:
I'm not sure if this is the right place to init Sentry in this extension.  Can you please confirm?

Reply:
it looks to be the right place.. in essence, every time a notification is received, sentry gets initialized.